src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1708,6 +1708,9 @@ def index(self) -> Union["pd.Index", native_pd.MultiIndex]:
         """
         if self.is_multiindex():
             # Lazy multiindex is not supported
+            logging.warning(
+                "Lazy MultiIndex is not supported. Pulling values from Snowflake."
```

Comment:
It's not clear if it is only multi-index values being pulled out, or entire dataframe is pulled out.
```suggestion
                "Lazy MultiIndex is not supported. MultiIndex values are evaluated eagerly and pulled out of Snowflake."
```